src/Summary.jsx

```diff
@@ -37,15 +37,15 @@ class Summary extends React.Component {
 
   componentDidMount() {
     const { activeBlock } = this.props;
-    const request = activeBlock ? RestAPI.requestGeometryByArea(activeBlock.id) : RestAPI.requestGeometryByArea('LLA 0970');
+    const blockId = (activeBlock && activeBlock.id) ? activeBlock.id : 'LLA 0970';
     this.setState({
       activeBlock,
       layers: {
         dagma: {
           displayName: 'dagma',
           id: 1,
           active: true,
-          layer: L.geoJSON(request, {
+          layer: L.geoJSON(RestAPI.requestGeometryByArea(blockId), {
```

Comment:
RestAPI.requestGeometryByArea(blockId) will return a promise (when implemented the request to the backend). So better keep the call outside the setState. Later the setState will be inside the promise resolution

Reply:
Done